<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>edit_agilent_config_dialog</class>
 <widget class="QDialog" name="edit_agilent_config_dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>391</width>
    <height>279</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Параметры Agilent3458A</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resources/icons.qrc">
    <normaloff>:/icons/icons/settings.png</normaloff>:/icons/icons/settings.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Способ подключения</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>COM name</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Порт</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Адрес GPIB</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>IP адрес</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Индекс GPIB</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="connect_type_combobox">
       <item>
        <property name="text">
         <string>Agilent USB-GPIB</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>NI USB-GPIB</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Prologix COM-GPIB</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Prologix Ethernet-GPIB</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QSpinBox" name="gpib_index_spinbox">
       <property name="maximum">
        <number>9999</number>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QSpinBox" name="gpib_address_spinbox">
       <property name="maximum">
        <number>9999</number>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="com_name_edit"/>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="ip_address_edit"/>
     </item>
     <item row="5" column="1">
      <widget class="QSpinBox" name="port_spinbox">
       <property name="maximum">
        <number>9999</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="buttons_layout">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="accept_button">
       <property name="text">
        <string>Принять</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancel_button">
       <property name="text">
        <string>Отмена</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>accept_button</tabstop>
  <tabstop>cancel_button</tabstop>
 </tabstops>
 <resources>
  <include location="../resources/icons.qrc"/>
 </resources>
 <connections/>
</ui>
